src/components/OptionsPage/Checkbox/LabeledCheckbox.tsx

```diff
@@ -48,7 +55,18 @@ export const LabeledCheckbox: FC<{
         value={props.value}
       />
 
-      <Paragraph>{props.description}</Paragraph>
+      <Paragraph>
+        {props.learnMoreLink === undefined ? (
+          props.description
+        ) : (
+          <>
+            {props.description}{' '}
+            <a className={paragraph} href={props.learnMoreLink} rel="noreferrer noopener" target="_blank">
+              {t('optionsPage_optionControl_learnMoreLink')}
+            </a>
+          </>
+        )}
+      </Paragraph>
```

Comment:
This could be simplified:

```jsx
<Paragraph>
  {props.description}

  {props.learnMoreLink !== undefined && (
    <a href={props.learnMoreLink} rel="noreferrer noopener" target="_blank">
      {t('optionsPage_optionControl_learnMoreLink')}
    </a>
  )}
</Paragraph>
```